tests/test_metamod.py

```diff
@@ -1,6 +1,70 @@
 import os
+import subprocess
 from pathlib import Path
 
+import pytest
+from imod.couplers.metamod import MetaMod
+from imod.mf6 import Modflow6Simulation, open_cbc, open_hds
+from imod.msw import MetaSwapModel
+from numpy.testing import assert_array_almost_equal
+
+
+@pytest.fixture
+def metamod_model_sprinkling(
+    coupled_mf6_model: Modflow6Simulation,
+    msw_model: MetaSwapModel,
+    metaswap_lookup_table: Path,
+) -> MetaMod:
+
+    # Override unsat_svat_path with path from .env
```

Comment:
```suggestion
    # Override unsat_svat_path with path from environment
```

The .env just sets the environment